### PR TITLE
Simple test for Owner Sign In

### PR DESCRIPTION
**Pull request by Zeicks**

### CODE REVIEW

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    private SignInPage signInPage;
+    private SignInFacade signInFacade;
+
+    @Test
+    public void signInTestsOwner() {
+        signInPage = new SignInPage(driver);
+        signInFacade = new SignInFacade(driver);
+        signInPage.goToSignInPage();
+        signInFacade.signIn(ownerEmail, ownerPassword);
+        Boolean element = (new WebDriverWait(driver, Duration.ofSeconds(4)).until(ExpectedConditions.urlToBe("http://localhost:3000/profile/restaurants")));
```
Comment (dmatsev) [inline src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: it should be a part of Page object

Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: update

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    public void signInTestsOwner() {
+        signInPage = new SignInPage(driver);
+        signInFacade = new SignInFacade(driver);
+        ownerPanel = new OwnerPanel(driver);
+        signInPage.goToSignInPage();
+        signInFacade.signIn(ownerEmail, ownerPassword);
+        ownerPanel.waitForPagePresence();
+        Assert.assertEquals(driver.getCurrentUrl(), ConfigProvider.ownerPageUrl, "Failed to login");
+    }
+}
```
Comment (vadymtsehelny) [inline src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: Can not run this test, compile error
java: cannot find symbol
  symbol:   class BaseComponent
  location: package com.easyrest.components

File: src/main/java/com/easyrest/pages/OwnerPanel.java [hunk 3]
```diff
@@ -31,4 +35,9 @@ public OwnerPanel(WebDriver driver) {
         administrators = new Administrators(driver);
         waiters = new Waiters(driver);
     }
+
+    public void waitForPagePresence() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/pages/OwnerPanel.java]: Hard code

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java (hunk 2 above)
Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: in the 35th pull request, I deleted this line, update your main

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import com.easyrest.pages.SignInPage;
+import com.easyrest.tests.BaseTest;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class OwnerSignIn extends BaseTest {
```
Comment (vadymtsehelny) [inline src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: Naming convention

File: src/main/java/com/easyrest/pages/OwnerPanel.java (hunk 3 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: Fixed

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java (hunk 4 above)
Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: fix

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java (hunk 2 above)
Comment (vadymtsehelny) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: So why did you remove that line 12 minutes ago if you did it in the 35th pull request?

Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: I don't know why they reappeared

File: src/main/java/com/easyrest/pages/OwnerPanel.java (hunk 3 above)
Comment (vadymtsehelny) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: I meant seconds) we have this data in constants package, class TimeoutVariable. I recommend taking the task that we have planned Investigate code in project.

**Review by vadymtsehelny — APPROVED** [review]

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java (hunk 2 above)
Comment (dmatsev) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: @vadymtsehelny @Zeicks, what is the current status of that issue?

File: src/main/java/com/easyrest/pages/OwnerPanel.java (hunk 3 above)
Comment (dmatsev) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: @Zeicks How did you resolve the comment?

File: src/test/java/com/easyrest/tests/owner/OwnerSignIn.java (hunk 2 above)
Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/OwnerSignIn.java]: Resolve

File: src/main/java/com/easyrest/pages/OwnerPanel.java (hunk 3 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: > I meant seconds) we have this data in constants package, class TimeoutVariable. I recommend taking the task that we have planned Investigate code in project.

Fixed

Comment (Zeicks) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: > @Zeicks How did you resolve the comment?
I thought the problem was the locator

**Review by dmatsev — APPROVED** [review]

**Review by martadiachok — APPROVED** [review]

**Pull request merged**